packages/staking-contracts/src/rewards-allocator.ts

```diff
@@ -93,6 +98,7 @@ class RewardsAllocator {
 
     const receipt = await this.publicClient.waitForTransactionReceipt({
       hash,
+      timeout: this.timeout,
```

Comment:
`viem`'s typedocs state the `pollingInterval` for `waitForTransactionReceipt` will default to whatever the `publicClient` or `walletClient` set. the `timeout`, however, needs to be explicitly set in this method.